### ISSUE TITLE
loss and val_loss decreasing

### ISSUE THREAD

**Issue by ayatirfann** [keras-team/keras#19658]
![los and val_loss](https://github.com/keras-team/keras/assets/122761782/9958a0e0-26d1-411d-a455-74bb24e0a341)
![accuracy and val_accuracy](https://github.com/keras-team/keras/assets/122761782/52eb1a77-bbec-4dd8-a074-74f384918fb3)
so I am building an image classification CNN model using TensorFlows sequential API. 

I don't know why my graphs are acting this way and why the training accuracy of my model isn't steadily going up. can anyone help?

**Comment by SuryanarayanaY** [comment]
Hi @ayatirfann ,

Your model loss and val_loss decreasing gradually but accuracy staying constant means something wrong. Not sure how the model compiled. Providing code snippet may help us. Thanks.
